### PR TITLE
Followup fixes after Quo integration

### PR DESCRIPTION
**Pull request by Ferossgp**

### CODE REVIEW

**Comment by Serhy** [comment]
Tested on Android 8, Android 9, iOS13. 
Works perfectly for me now. Long taps upon chats on chats list or browser history items are working for me. 
Password with 'enabled-eye' icon revealing typed chars fixed as well.

File: src/quo/components/list/item.cljs [hunk 1]
```diff
@@ -59,7 +59,7 @@
                                 {:min-height       (size->container-size size)
                                  :padding-vertical 8
                                  :flex-direction   :row
-                                 :flex             1
+                                 ;; :flex             1
```
Comment (Ferossgp) [inline src/quo/components/list/item.cljs]:
```suggestion
```

File: src/quo/components/text_input.cljs [hunk 2]
```diff
@@ -178,8 +178,9 @@
                             (and platform/ios? (= keyboard-type "visible-password"))
                             "default"
 
-                            (and platform/android? secure-text-entry @visible)
-                            "visible-password"
+                            ;; NOTE: Now switching dynamically brakes android input
+                            ;; (and platform/android? secure-text-entry @visible)
+                            ;; "visible-password"
```
Comment (flexsurfer) [inline src/quo/components/text_input.cljs]: in this case, it might be saved by OS and used in suggestions like regular text ?

Comment (Ferossgp) [reply src/quo/components/text_input.cljs]: Yes, it will be a regular text. Not sure how OS stores it

Comment (Ferossgp) [reply src/quo/components/text_input.cljs]: Found another fix, will push soon

Comment (Ferossgp) [reply src/quo/components/text_input.cljs]: done

**Review by flexsurfer — APPROVED** [review]

**Comment by Serhy** [comment]
Tested password(s) on Android8, 9, iOS13, and long taps are still working perfectly for me.

**Pull request merged**